Pyfhel/Afhel/Afseal.h

```diff
@@ -228,6 +229,7 @@ class Afseal: public Afhel {
 
   std::shared_ptr<seal::SEALContext> context = NULL;     /**< Context. Used for init*/
   std::shared_ptr<seal::BatchEncoder> bfvEncoder = NULL; /**< Rotation in Batching. */
+  std::shared_ptr<seal::BatchEncoder> bgvEncoder = NULL; /**< Rotation in Batching. */

```

Comment:
Remove the "Rotation in Batching from all three encoders, as it serves no purpose now.